tests/engines/collections/acceptance/submit/submit-test.ts

```diff
@@ -0,0 +1,64 @@
+import { click as untrackedClick } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { percySnapshot } from 'ember-percy';
+import { module, test } from 'qunit';
+
+import { Permission } from 'ember-osf-web/models/osf-model';
+import { click, visit } from 'ember-osf-web/tests/helpers';
+import { setupEngineApplicationTest } from 'ember-osf-web/tests/helpers/engines';
+
+module('Collections | Acceptance | submit', hooks => {
+    setupEngineApplicationTest(hooks, 'collections');
+    setupMirage(hooks);
+
+    test('it renders', async () => {
+        server.loadFixtures('taxonomies');
+        server.loadFixtures('licenses');
+        const currentUser = server.create('user', 'loggedIn');
+        const primaryCollection = server.create('collection');
+        const nodeToBeAdded = server.create('node', {
+            title: 'Node to be added to collection',
+            currentUserPermissions: Object.values(Permission),
+        });
+        server.create('contributor', {
+            node: nodeToBeAdded,
+            users: currentUser,
+            index: 0,
+        });
+        const taxonomies = server.schema.taxonomies.all().models;
+        const licensesAcceptable = server.schema.licenses.all().models;
+        const provider = server.create('collection-provider', {
+            id: 'studyswap',
+            primaryCollection,
+            taxonomies,
+            licensesAcceptable,
+        });
+        await visit(`/collections/${provider.id}/submit`);
+        await percySnapshot('Collections | Acceptance | submit | select project');
+        await untrackedClick('[data-test-collections-item-picker] .ember-power-select-trigger');
+        await untrackedClick(`[data-test-collections-node-title="${nodeToBeAdded.title}"]`);
+        await percySnapshot('Collections | Acceptance | submit | project metadata');
+        await untrackedClick('[data-test-collections-license-picker] .ember-power-select-trigger');
+        await untrackedClick('.ember-power-select-option');
+        await untrackedClick('[data-test-project-metadata-continue]');
+        await percySnapshot('Collections | Acceptance | submit | project contributors');
+        await untrackedClick('.btn-primary');
```

Comment:
Change this to use a test selector also, please. You can put a generic test selecor on the save button in the `submit-section-buttons` component and a specific test selector on the `project-contributors` component and do:
```suggestion
        await untrackedClick('[data-test-project-contributors] [data-test-save-button]');
```